.yarnrc.yml

```diff
@@ -1,7 +1,7 @@
-yarnPath: .yarn/releases/yarn-4.7.0.cjs
-
-nodeLinker: node-modules
+defaultSemverRangePrefix: ""
 
 nmMode: hardlinks-global
 
-defaultSemverRangePrefix: ""
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.9.1.cjs
```

Comment:
The order of the lines in this file changed when running `yarn set version 4.9.1`